conf/summariser.cfg

```diff
@@ -6,3 +6,20 @@ pidfile = /var/run/apel/summariser.pid
 logfile = /var/log/apel/summariser.log
 level = INFO
 console = true
+
+# Define if and how the summariser should handle stale summaries.
+[stale_summaries]
+# If enable_clean_up equals true, stale summaries in the database will
+# be deleted.
+enable_clean_up = true
+# Use newer_than to limit the deletion of stale summaries to those
+# reporting recent usage. This avoids deleting stale summaries that
+# may be produced by deleting individual records regularly as part an
+# implementation of a long term retention policy.
+# The recommended setting is that newer_than should be set to twice the time
+# between individual summariser runs to ensure the month boundary is handled
+# correctly.
+# i.e. 2 days if you summarise daily.
+# The unit of newer_than is days, though the cut off used will be rounded down
+# to the start of the calendar month.
+newer_than = 2
```

Comment:
`stale_summary_window_days` ?

Reply:
Addressed in [68e12d7](https://github.com/apel/apel/pull/222/commits/68e12d71abc6892e9b4a8ad3b2c79b2426ca068d), but not yet tested.